src/julia-parser.scm

```diff
@@ -1272,7 +1272,7 @@
                                        (parse-raw-literal s t)))
                         (nxt (peek-token s))
                         (macname (macroify-name ex (macsuffix t))))
-                   (if (and (symbol? nxt) (not (operator? nxt))
+                   (if (and (or (symbol? nxt) (number? nxt)) (not (operator? nxt))
                             (not (ts:space? s)))
                        ;; string literal suffix, "s"x
                        (loop `(macrocall ,macname ,startloc ,macstr
```

Comment:
If the suffix is a number, maybe it should be left as-is instead of converting to a string?

Reply:
My Scheme skills weren't quite up to that (rusty after 40 years!).
How would that be done?
I handled it in the macro `ss` (already merged into ShortStrings.jl, hoping that this change would make it into 1.6 for the next LTS release), by calling `parse`, it still happens at compile-time, but yes, having a number would be nicer.